webapp/src/reducers.ts

```diff
@@ -226,9 +226,9 @@ interface userReactionsState {
 }
 
 const queueReactions = (state: ReactionWithUser[], reaction: ReactionWithUser) => {
-    const result = [...state];
+    const result = state?.length ? [...state] : [];
     result.push(reaction);
-    if (result.length > 8) { // TODO: random size, this should probably be configurable
+    if (result.length > 50) { // TODO: random size, this should probably be configurable
```

Comment:
Is this for debugging purposes? Or do we want 50?

Reply:
I wanted a big number enought to cover most screens, originally we agreed on 8 but later on decided on 70% of the screen, but I don't want to hold the entire history of reactions, as a two hour call might be able to crash lower end computers, so either 50 or whatever people feel it is a good number, but not limitless :)